src/query/src/datafusion.rs

```diff
@@ -212,10 +218,19 @@ impl DatafusionQueryEngine {
         column_vectors: HashMap<String, VectorRef>,
         query_ctx: QueryContextRef,
     ) -> Result<Output> {
+        let catalog_name = table_name.catalog.to_string();
+        let schema_name = table_name.schema.to_string();
+        let table_name = table_name.table.to_string();
+
+        ensure!(
+            schema_name != "information_schema",
```

Comment:
Instead of this constant, I'd like to add a read-only schema test function in `catalog` crate, then we can check it by:

```
ensure!(!catalog::consts::is_readonly_schema(schema_name), TableReadOnlySnafu...);
```

If we add other read-only schemas, we can just change it in the `is_readonly_schema` function.